app/controllers/gobierto_people/people/google_calendar/authorization_controller.rb

```diff
@@ -42,7 +42,8 @@ def load_person
         end
 
         def load_google_calendar_configuration
-          @configuration = ::GobiertoPeople::PersonGoogleCalendarConfiguration.find_or_initialize_by person_id: @person.id
+          @configuration = ::GobiertoCalendars::GoogleCalendarConfiguration.find_by(collection_id: @person.calendar.id)
```

Comment:
This configuration might not exist. See this Rollbar: https://rollbar.com/Populate/gobierto/items/646/